docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
```

Comment:
Does it make sense to have an `experimental` namespace like we do in `pydantic` for things like this that might explicitly change?

Reply:
Answered in person.